### DISCUSSION TITLE
Reg WIX Binaries usage from Jpackage

### DISCUSSION THREAD

**Discussion by pari205** [wixtoolset/issues#7841] [Questions]
Hi Team,

Thanks for providing WIX binaries. We have few questions around the way WIX is used by jpackage utility provided by JDK.

When jpackage is used in windows along with WIX to covert jar into executable windows installer,

-     What are the exe used by jpackage from WIX binaries?
-     While packaging the installer, any files from WIX is copied to the exe?

Could you provide insights around this?

Regards
Pari.V

**Comment by chrpai** [comment]
This was asked two days ago by someone else.  Is there a related inquiry?

https://github.com/orgs/wixtoolset/discussions/7838

**Reply by pari205** [reply]
Yes @chrpai Similar query , we are using jpackage which internally uses WIX binaries(added in the path), not using any other utility.

**Comment by chrpai** [comment]
There's a number of tools like this.  Electron has one for example.  I'm not really versed on them because they all solve a nich authoring problem and I focus on authoring installers for any kind of application using any stack. It just not worth my time to investigate how each of these solutions have uniquely attempted to abstract WiX away when I'm already a WiX and InstallShield expert.  My answer in the other thread is likely the right answer.  Consulting services are available if you'd like a deeper analysis.
